components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -498,6 +498,9 @@ protected void processAuthenticationResponse(HttpServletRequest request,
                                                  HttpServletResponse response, AuthenticationContext context)
             throws AuthenticationFailedException {
 
+        context.setProperty(BasicAuthenticatorConstants.CAPTCHA_PARAMS,
+                getCaptchaParams(context.getLoginTenantDomain(), 0));
```

Comment:
better to set this to context only if `getCaptchaParams(context.getLoginTenantDomain(), 0)` not empty string as this is not needed to be set if recaptcha is disabled

Reply:
Addressed in 0368728cbbd6a6d3edf8010df68c76ef4b9777ad